card/mod/05_standard/set/abstract/attachment.rb

```diff
@@ -56,7 +56,7 @@ def assign_attachment file, original_filename
 end
 
 def file_ready_to_save?
-  attachment.file &&
+  attachment.file.present? &&
```

Comment:
`attachment.file` always exists. The `present?` is overridden to check if any file inside the `file`.